staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_external.go

```diff
@@ -1045,27 +1035,17 @@ func ensureStaticIP(s CloudAddressService, name, serviceName, region, existingIP
 	desc := makeServiceDescription(serviceName)
 
 	var creationErr error
-	switch netTier {
-	case cloud.NetworkTierPremium:
-		addressObj := &compute.Address{
-			Name:        name,
-			Description: desc,
-		}
-		if existingIP != "" {
-			addressObj.Address = existingIP
-		}
-		creationErr = s.ReserveRegionAddress(addressObj, region)
-	default:
-		addressObj := &computealpha.Address{
-			Name:        name,
-			Description: desc,
-			NetworkTier: netTier.ToGCEValue(),
-		}
-		if existingIP != "" {
-			addressObj.Address = existingIP
-		}
-		creationErr = s.ReserveAlphaRegionAddress(addressObj, region)
+	addressObj := &compute.Address{
+		Name:        name,
+		Description: desc,
+	}
+	if existingIP != "" {
+		addressObj.Address = existingIP
+	}
+	if netTier != "" {
```

Comment:
Can we be consistent. Either use this method in the object construction or do it after in this sort of if statement. 
I would lean toward just doing it in object construction since ToGCEValue() will return empty string if called on an unsett netTier.

Reply:
Thanks for spotting this. I fixed it now 👍